lib/smooth/application/display/DisplaySpi.cpp

```diff
@@ -59,87 +58,146 @@ namespace smooth::application::display
                         use_pre_transaction_callback,
                         use_post_transaction_callback),
 
-              reset_pin(reset_pin, true, false, false),
-              backlight_pin(back_light_pin, true, false, false),
-              dc_pin(data_command_pin, true, false, false),
-              cs_pin(chip_select_pin, true, false, false)
+              reset_pin(reset_pin, true, false, false),     // GPIO_MODE_OUTPUT, no pullup, no pulldown
+              dc_pin(data_command_pin, true, false, false), // GPIO_MODE_OUTPUT, no pullup, no pulldown
+              cs_pin(chip_select_pin, true, false, false)   // GPIO_MODE_OUTPUT, no pullup, no pulldown
```

Comment:
Is this hardware dependent, i.e. does it makes sense to have these settings passed in to the class in the constructor?

Reply:
All the display chips I have looked at require chip select (cs), data/command (dc) and possibly reset.  The constructor currently takes the gpio_num_t for these three pins.  I guess I am not sure what you are asking or looking for?

Reply:
My bad, ignore :)